src/main/java/se/lth/cs/connect/routes/Admin.java

```diff
@@ -88,6 +89,21 @@ protected void setup(PippoSettings conf) {
             });
             rc.getResponse().ok();
         });
+        
+        
+        POST("/delete-collection", (rc) -> {
+            if (rc.getParameter("id").isEmpty())
+                throw new RequestException("Must provide entry parameter");
+            
+            int id = rc.getParameter("id").toInt();
+            final JcNode c = new JcNode("c");
+            Database.query(rc.getLocal("db"), new IClause[]{
+            	MATCH.node(c).label("collection"),
+            	WHERE.valueOf(c.id()).EQUALS(id),
+            	DO.DETACH_DELETE(c)
```

Comment:
This is insufficient - entries orphaned due to the detach should also be removed. Maybe we should add some kind of event triggered system and collect all these "on-entry-removal" and "on-collection-removal" events there.